docs/content/en/docs/Cluster operator/quickstart.md

```diff
@@ -79,22 +79,47 @@ Create a kubernetes cluster on AWS, which contains control plane nodes and worke
     link="./image/capa-aws.png"
     >}}
 
+Generating the cluster configuration:
+
+```console
+export AWS_CONTROL_PLANE_MACHINE_TYPE=t3.large
+export AWS_NODE_MACHINE_TYPE=t3.large
+export AWS_REGION=us-east-1
+export AWS_SSH_KEY_NAME=default
+export KUBERNETES_VERSION=v1.25.0
+export CONTROL_PLANE_MACHINE_COUNT=3
+export WORKER_MACHINE_COUNT=3
+clusterctl generate cluster capi-quickstart --infrastructure=aws:v2.0.0 > manifests/examples/capi-quickstart.yaml
```

Comment:
How do you fix it can not gen yaml with cluster operator?

Reply:
use `--infrastructure=aws:v2.0.0` with specific provider instead of default provider.